### ISSUE TITLE
References section of chutzpah.json being ignored

### ISSUE THREAD

**Issue by MrYossu** [mmanela/chutzpah#660]
I'm using Jasmine to unit test some JavaScript in a Visual Studio MVC project, using the Chutzpah Visual Studio extension. The test files live in a separate project.

I have the following lines at the top of a test file...

```
///<reference path="../../KendoMvvm/Scripts/jQuery/jquery-1.10.2.min.js"/>
///<reference path="../../KendoMvvm/Scripts/Kendo/kendo.all.min.js"/>
///<reference path="../../KendoMvvm/Scripts/Utils.js"/>
///<reference path="../../KendoMvvm/Scripts/Home/Grid.js"/>
```

...where the relative paths point to the files in the MVC project. Set up like this, the tests all run fine.

If I add a chutzpah.json file to the project, with the following content...

```
{
  "References": [
    { "Path": "../KendoMvvm/Scripts/jQuery/jquery-1.10.2.min.js" },
    { "Path": "../KendoMvvm/Scripts/Kendo/kendo.all.min.js"}
  ]
}
```

...and remove the first two lines from the test file, then the tests all fail, seemingly because it hasn't loaded jQuery and Kendo, so can't produce the objects needed. Note that I changed the relative path in the chutzpah.json file, as this lives at the root of the test project, whereas the test file lives in a first-level folder.

It looks like Chutzpah is just ignoring this file. I added a setting to change the framework to QUnit, which should have failed (as I'm using the MS test runner not QUnit), but the tests ran as before.

What am I doing wrong? According to the [Chutzpah docs](https://github.com/mmanela/chutzpah/wiki/references-setting), this should work.

**Comment by mmanela** [comment]
I suggest you run the chutzpah with the /trace argument if using the command line (you can also turn this on in the VS settings in which it should log to %appdata%). This will log everything chutzpah tried to do and tell you where it looked for and couldnt find that file.

**Comment by MrYossu** [comment]
Hello,

I switched on tracing, but can't find the log file. %appdata% just contains the three folders Local, LocalLow and Roaming, and I can't see any Chutzpah-related folders or files under any of those.

Please advise.
Thanks for the reply.

**Comment by mmanela** [comment]
Oh, sorry. The log goes to %temp% called chutzpah.log

**Comment by MrYossu** [comment]
Where? In C:\Users\MyUser\AppData itself, or in a subfolder?

Either way, I can't see one at all. Tried searching C:\Users\MyUser\AppData and subfolders for chutzpah.log but it didn't find anything. I did enable Chutzpah tracing from the VS Tools -> Options -> Chutzpah section.

Any ideas? Thanks again.

**Comment by MrYossu** [comment]
P.S. Forgot to add these in case they are relevant...

VS 2015 Enterprise Update 3
Chutzpah  Test Adapter for the Test Explorer v4.3.4

**Comment by PandaWood** [comment]
Actually, looks like this is the same issue that I just raised https://github.com/mmanela/chutzpah/issues/662

Though I used chutzpah_reference and I haven't tried throwing all the references into the file and removing the "Reference" section in config - to see if that works.

But I can check the trace/debug output and confirm that the chutzpah_reference is not being included or attempted in any way... there is no mention of the referenced file in the trace file. It's ignored.
